stablehlo/tests/interpret_constant.mlir

```diff
@@ -0,0 +1,95 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
```

Comment:
Can we use the same order of tupes as used in interpret_add.mlir ? First floating point types in increasing order of bitwidth and then integer with increasing order of bitwidth, and lastly the complex types? Just for consistency.

Reply:
Speaking of consistency, should we follow the order--integer, boolean, float, and complex--to match how the spec types are ordered? I will make another PR to fix `interpret_add.mlir` file along with any typos/grammar errors in the `spec_draft.md`.

Reply:
That should be fine as long as we stick to the same order. 
>>  I will make another PR to fix interpret_add.mlir file 
Makes sense. In this PR we could include all the tests (maybe using the same constant as in add test) and  a follow up PR can maintain the order for-all tests.